napari/_qt/layer_controls/_tests/test_qt_layer_controls.py

```diff
@@ -178,11 +178,16 @@ def test_create_layer_controls(
             qcombobox.setCurrentIndex(qcombobox_initial_idx)
 
 
-if sys.version_info[:2] == (3, 11) and qtpy.API == 'pyqt5':
+if (
+    sys.version_info[:2] == (3, 11)
+    and qtpy.API == 'pyqt5'
+    or qtpy.API == 'pyqt6'
```

Comment:
```suggestion
    sys.version_info[:2] == (3, 11)
    and (qtpy.API == 'pyqt5' or qtpy.API == 'pyqt6')
```

as 
```python
In [1]: False and False or True
Out[1]: True

In [2]: False and (False or True)
Out[2]: False
```

So this clause disables it on any pyqt6 test, not only python 3.11.

Reply:
Christ, sorry about that. 😭

Reply:
Operator precedence is not simple and it is easy to do some mistake.